jobs/fxci-taskcluster-export/fxci_etl/pulse/handler.py

```diff
@@ -91,8 +91,14 @@ def __init__(self, config: Config, **kwargs: Any):
 
     def process_event(self, event):
         data = event.data
+
+        if data.get("runId") is None:
+            # This can happen if `deadline` was exceeded before a run could
```

Comment:
To be clear: we can still have deadline exceeded with a `runId` (in which case we still process it, which is what we want)?

Reply:
I'm not sure. I didn't come across this case in the wild, but I noticed that the TC docs mentioned that `runId` might not always be set and gave `deadline-exceeded` as an example of when that would happen.

For now, I think it's fine to ignore events where it's unset. Tasks that never ran don't have a cost.

Reply:
I certainly agree that it's safe to ignore anything without a `runId`, for the record! This was mostly a note that `deadline-exceeded` may also happen when a `runId` is present, AIUI.